packages/common-types/src/accountHistory.ts

```diff
@@ -163,42 +162,37 @@ export interface SwapApiEvent {
 }
 
 export interface EarnDepositApiDataEvent {
-  strategyId: StrategyId;
-  positionId: SdkEarnPositionId;
-  asset: {
-    amount: string;
-    address: Address;
-    price?: number;
-  };
+  owner: Address;
+  permissions: Record<Lowercase<Address>, ('WITHDRAW' | 'INCREASE')[]>;
+  assetPrice?: number;
+  assetsDeposited: string;
+  depositToken: Address;
+  depositAmount: string;
 }
 export interface EarnDepositApiEvent {
   data: EarnDepositApiDataEvent;
   type: TransactionEventTypes.EARN_CREATED;
 }
 
 export interface EarnIncreaseApiDataEvent {
-  strategyId: StrategyId;
-  positionId: SdkEarnPositionId;
-  asset: {
-    amount: string;
-    address: Address;
-    price?: number;
-  };
+  assetPrice?: number;
+  assetsDeposited: string;
```

Comment:
can it be several or is it only one?

Reply:
Only one. It's the amount expressed in asset token

Reply:
then it should be assetDepositedAmount :)

Reply:
They use that naming for their contracts. I will change it for our parsed events, but the api would not change from their side

Reply:
then don't change it on our side